composer.json

```diff
@@ -90,7 +90,7 @@
         "nnnick/chartjs": "^3.9",
         "npm-asset/jquery-ui-touch-punch": "^0.2.3",
         "oomphinc/composer-installers-extender": "^2.0",
-        "rhodeislandecms/ecms_profile": "1.0.5",
+        "rhodeislandecms/ecms_profile": "dev-RIGA-526/patch-update",
```

Comment:
This needs to be updated to include this PR https://github.com/State-of-Rhode-Island-eCMS/ecms_profile/pull/512 before we can approve.

Reply:
@montgomeryyoung, I think this PR looks good. Just need to update the profile reference.

Reply:
I updated the reference to the profile rc branch which will include the PR